com.musala.simple.students.db/src/main/java/com/musala/simple/students/db/Main.java

```diff
@@ -0,0 +1,78 @@
+package com.musala.simple.students.db;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.log4j.BasicConfigurator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+import com.musala.simple.students.db.exception.StudentNotFoundException;
+import com.musala.simple.students.db.helpers.FileHelper;
+import com.musala.simple.students.db.helpers.ValidationHelper;
+import com.musala.simple.students.db.internal.ErrorMessage;
+import com.musala.simple.students.db.student.Student;
+import com.musala.simple.students.db.student.StudentDataPrinter;
+import com.musala.simple.students.db.student.StudentGroup;
+import com.musala.simple.students.db.student.StudentWrapper;
+
+public class Main {
+	public static void main(String[] args) {
+		Logger logger = LoggerFactory.getLogger(Main.class);
+		BasicConfigurator.configure();
```

Comment:
What does log4j has to do with slf4j? Do we need this statement and `import org.apache.log4j.BasicConfigurator;` at all?